From 25d66ccc033ebc35cebad38131361ad93f570e0d Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Mon, 18 Jun 2018 14:55:14 +0900
Subject: [PATCH 3970/5242] MLK-18625-2 include: dts: imx8mq clk rename
 external pll source

commit  2fcdb3b589f163ad9d6c6c83586e466ffb5e96cd from
https://source.codeaurora.org/external/imx/linux-imx.git

External differential clock phy_27m can be set to all
plls, rename from VIDEO2_PHY_27M to CLK_PHY_27M to avoid
confusion as clock source is the same option for all plls

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
(cherry picked from commit e4ac6dff8fa2eda6f5c2ed35cfea3550c59916da)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/dt-bindings/clock/imx8mq-clock.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dt-bindings/clock/imx8mq-clock.h b/include/dt-bindings/clock/imx8mq-clock.h
index 0daffb5..467abec 100644
--- a/include/dt-bindings/clock/imx8mq-clock.h
+++ b/include/dt-bindings/clock/imx8mq-clock.h
@@ -628,7 +628,7 @@
 
 #define IMX8MQ_CLK_NAND_USDHC_BUS_RAWNAND_CLK	488
 
-#define IMX8MQ_VIDEO2_PHY_27MHZ			489
+#define IMX8MQ_CLK_PHY_27MHZ			489
 
 #define IMX8MQ_CLK_END                         490
 
-- 
1.7.9.5

